### ISSUE TITLE
Cleanup the code regarding the cursor sound

### ISSUE THREAD

**Issue by LevFendi** [Factorio-Access/FactorioAccess#176]
The close inventory sound effect (which is real but also pretty quiet) is used extensively as the cursor sound. It is called with the api function play_sound every time, but with some extra checks. It should really get its own function so that we can use a better name and boost the volume and do other things without duplicating code.

**Comment by ahicks92** [comment]
I upped all my volumes.  There's an issue for changing the defaults.  Whatever this sound is supposed to be I cannot hear it even when moving around, even with the volume settings I changed.  Maybe moving it oiut to our own file will fix it but ultimately I don't think quiet is the problem.

if we do move it out, let's start putting these behind functions so thate we can change the properties globally and change the files without finding all the strings.  Probably not hard to do the rest, just time consuming.  Since we are putting credits directly in our sound's filenames, I do think that's apropos.

**Comment by LevFendi** [comment]
It is rather odd that you do not hear this particular sound. To test, it is most obvious when you run along a transport belt in smooth walk mode, because it plays every time the cursor hits an entity.

**Comment by ahicks92** [comment]
Yeah ok so I do hear that one.  Is it supposed to play when in cursor mode? That is possibly the point of confusion.

**Comment by LevFendi** [comment]
Yes and the source of the confusion is issue #180, which i had forgotten to document.
